script/Ciel/Day 1/01_01_CIEL01_3_3.txt

```diff
@@ -211,17 +219,19 @@ The back of my neck turns red at her teasing voice.
 [sha:ba4b3509fd47e92e1e33cb04fdcb41250f410b09]{
 -- Page 205, Offset 975.
 -- シエル先輩に指摘されるのも無理はない。
-It's no wonder that Ciel-senpai pointed it out.
+It's no wonder that Ciel-senpai called me out on it.
 }
 [sha:da7a3da6005c8d8140b615f6e3fafd6fddc5be28]{
 -- Page 205, Offset 976.
 -- 朝に“興味があるなら茶道室まで”と言われたから来るなんて、普段の自分からは考えられないコトだ。
-She only asked me to come if I was interested, which is something I wouldn't normally do.
+She only asked me to come if I was interested,
+ and normally I wouldn't even think about following through on something like this.
 }
 [sha:409ddfefbf3155dd944e0cc98a7df51716287cc6]{
 -- Page 205, Offset 977.
 -- 「……と、これは失礼。強くかけすぎちゃいましたか」
-"Oh, I'm sorry. I think I put too much pressure on you."
+// Presumably talking about overdoing the suggestion or sth
+"Oh, I'm sorry. Maybe I came on a little too strong."
```

Comment:
I think she means she teased him too hard

Reply:
There's a weird flash here, which is why I assumed shenanigans

Reply:
Ah, playing the scene you're right. Yeah, she's fiddling with his head, lol.

Might want to make it a smidge more explicit then

>Oh, I'm sorry. Seems I went a bit overboard.